### PR TITLE
Ignore jscomp warnings in transpiled typescript

### PR DESCRIPTION
**Pull request by jart**
Please note that we still display errors. For example, an undeclared global variable is considered an error.

### CODE REVIEW

File: tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java [hunk 1]
```diff
@@ -391,6 +391,12 @@ public CheckLevel level(JSError error) {
             if (error.sourceName == null) {
               return null;
             }
+            if (error.getDefaultLevel() == CheckLevel.WARNING
+                && isErrorFromTranspiledTypescriptCode(error)) {
+              // Let's put our faith in the TypeScript compiler. At least until we have tsickle as
```
Comment (dandelionmane) [inline tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java]: nit: line length

File: tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java [hunk 2]
```diff
@@ -455,6 +461,17 @@ public CheckLevel level(JSError error) {
     verify(tags.isEmpty(), "<script> wasn't compiled: %s", tags);
   }
 
+  private static boolean isErrorFromTranspiledTypescriptCode(JSError error) {
+    // We perform this check by looking for a concomitant .d.ts webfile which is generated by the
+    // TypeScript compiler. Ideally we would use SourceExcerptProvider to determine the original
```
Comment (dandelionmane) [inline tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java]: nit: line length

File: tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java (hunk 1 above)
Comment (wchargin) [inline tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java]: Line length is 100 for Java.

Comment (jart) [inline tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java]: Indeed.

**Pull request merged**